### PR TITLE
update fapi-client library to 3.3.7

### PR DESCRIPTION
**Pull request by dskamiotis**
## What's changed?

This is a simple PR to update to `"com.gu" %% "fapi-client-play27" % "3.3.7",`.
This will allow the client to be able to add the Podcast item to the list of tags.
## Implementation notes

I significant jump from 3.2.0 to 3.3.7

This has been initially tested using SNAPSHOT locally and from facia-tool to ensure the logic change in the library works.

In addition:
-  CAPI libs bumped `capiModelsVersion = "17.1.0"`   `capiClientVersion = "17.21"`
-  Updates the models used in fronts to be in line with latest CAPI models. 
-  This results in all test passing

| Before  | After |
| --- | --- |
| image| image|

### General
- [ ] 🤖 Relevant tests added
- [x] ✅ CI checks / tests run locally
- [ ] 🔍 Checked on CODE

Tested by adding `Podcast` from tags dropdown in ` https://fronts.code.dev-gutools.co.uk/editorial/config`
### Client
- [x] 🚫 No obvious console errors on the client (i.e. React dev mode errors)
- [ ] 🎛️ No regressions with existing user interactions (i.e. all existing buttons, inputs etc. work)
- [ ] 📷 Screenshots / GIFs of relevant UI changes included

### CODE REVIEW

**Comment by Reettaphant** [comment]
Have you checked that you can still press the fronts after this change ie that frontend can still process any changes you make? To test this edit a front locally and add the new podcast tag on a container, save and launch changes and then check for changes here: http://m.code.dev-theguardian.com/{name-of-front} (more instructions in the Readme)

**Comment by dskamiotis** [comment]
> Have you checked that you can still press the fronts after this change ie that frontend can still process any changes you make? To test this edit a front locally and add the new podcast tag on a container, save and launch changes and then check for changes here: [http://m.code.dev-theguardian.com/{name-of-front}](http://m.code.dev-theguardian.com/%7Bname-of-front%7D) (more instructions in the Readme)

**Comment by dskamiotis** [comment]
Yes all working, I added the Podcast tag and checked pressed:
image

**Comment by Reettaphant** [comment]
Looks great, thank you!

File: scripts/setup.sh [hunk 1]
```diff
@@ -75,7 +75,7 @@ download_config() {
 
 fetch_config(){
     if [[ -w ${CONFIG_DIR} ]]; then
-        downloadConfig
+        download_config
```
Comment (jonathonherbert) [inline scripts/setup.sh:78]: Is this change necessary in this PR?

Comment (dskamiotis) [reply scripts/setup.sh:78]: I believe this is invoking the function which now lives in this file (since we changed the set up config in another PR).
I thought it may cause an error in CI/test builds if not amended. However seems to be working fine locally.

Comment (jonathonherbert) [reply scripts/setup.sh]: Ah, yeah – it may be working locally because there are other changes in your git tree that are untracked, or haven't been committed 👍

**Review by jonathonherbert — APPROVED** [review]
Change looks good, testing looks good – great work.
